src/templates/fr/2019/methodology.html

```diff
@@ -0,0 +1,724 @@
+{% extends "fr/2019/base.html" %}
+
+{% block title %}Méthodologie | Web Almanac par HTTP Archive{% endblock %}
+
+{% block description %}Décrit comment le Web
+Almanac 2019 a été conçu : Les ensembles de données et outils utilisés et la
+façon dont le projet a été exécuté.{% endblock %}
+
+{% block meta %}
+<meta name="description" content="{{ self.description() }}">
+
+<meta property="og:title" content="{{ self.title() }}">
+<meta property="og:url" content="https://almanac.httparchive.org{{  url_for(request.endpoint, **get_view_args(lang=language.lang_code, year=year)) }}">
+<meta property="og:image" content="https://almanac.httparchive.org/static/images/methodology-banner.png">
+<meta property="og:image:height" content="984">
+<meta property="og:image:width" content="1200">
+<meta property="og:type" content="article">
+<meta property="og:description" content="{{ self.description() }}">
+
+<meta name="twitter:card" content="summary_large_image">
+<meta name="twitter:site" content="@HTTPArchive">
+<meta name="twitter:title" content="{{ self.title() }}">
+<meta name="twitter:image" content="https://almanac.httparchive.org/static/images/methodology-banner.png">
+<meta name="twitter:image:alt" content="Méthodologie du Web Almanac {{ year }}" />
+<meta name="twitter:description" content="{{ self.description() }}">
+
+<script type="application/ld+json">
+  {
+    "@context": "http://schema.org",
+    "@type": "Article",
+    "mainEntityOfPage": {
+      "@type": "WebPage",
+      "@id": "https://almanac.httparchive.org{{  url_for(request.endpoint, year=year, lang=lang) }}"
+    },
+    "headline": "{{ self.title() }}",
+    "image": {
+      "@type": "ImageObject",
+      "url": "https://almanac.httparchive.org/static/images/methodology-banner.png",
+      "height": 984,
+      "width": 1200
+    },
+    "publisher": {
+      "@type": "Organization",
+      "name": "HTTP Archive",
+      "logo": {
+        "@type": "ImageObject",
+        "url": "https://almanac.httparchive.org/static/images/ha.png",
+        "height": 160,
+        "width": 320
+      }
+    },
+    "author": {
+      "@type": "Person",
+      "sameas": [
+        "https://almanac.httparchive.org{{ url_for('contributors', year=year, lang=lang, _anchor='rviscomi') }}",
+        "https://twitter.com/rick_viscomi",
+        "https://github.com/rviscomi"
+      ],
+      "name": "Rick Viscomi"
+    },
+    "description": "{{ self.description() }}",
+    "datePublished": "2019-11-04T12:00:00+00:00:00",
+    "dateModified": "2019-11-04T12:00:00+00:00:00"
+  }
+</script>
+<script type="application/ld+json">
+  {
+    "@context": "https://schema.org",
+    "@type": "BreadcrumbList",
+    "itemListElement": [
+      {
```

Comment:
Same again here (I actually prefer your spacing here to be honest, so happy to change English version instead if you prefer - as long as they are the same).